modules/ROOT/pages/working-with-environments.adoc

```diff
@@ -99,4 +96,5 @@ Please note that in this scenario, should the target environment P2 have _any_ o
 ==== Scenario 4: Conflicting Identifiers and Promotions
 * Suppose there's a partner P1 on the source environment E1. Suppose P1 has a set of identifiers (types and values) for AS2, EDI X12 and EDIFACT.
 * And suppose there is a partner P2 on some target environment E2. Suppose P2 has, for whatever reason, the exact same set of identifiers as P1 on E1
-* Promoting P1 from E1 to E2, while selecting "Keep Existing Identifiers", will try to "copy" P1 into E2. This will cause that a set of identifiers be created in E2. However, having the same identifier values more than once is illegal, so promotion will fail. The "Promotion" window will display a message informing about this situation.
\ No newline at end of file
+* Promoting P1 from E1 to E2, while selecting "Keep Existing Identifiers", will try to "copy" P1 into E2. This will cause that a set of identifiers be created in E2. However, having the same identifier values more than once is illegal, so promotion will fail. The "Promotion" window will display a message informing about this situation.
+////
```

Comment:
from line 20 to 100, everything is commented out. That doesn't seem to be the same as https://docs.mulesoft.com/partner-manager/1.0/working-with-environments. Can you confirm that you intend to comment all of them out?

Reply:
This file isn't referenced in the current latest index.adoc. I can only guess that it was a draft that was never removed. I only fixed the merge conflict.